doc/reference/configuration/configuration_reference.rst

```diff
@@ -11,6 +11,329 @@ This topic describes all :ref:`configuration parameters <configuration>` provide
+
+audit_log
+---------
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    Configuring ``audit_log`` parameters is available in the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+The ``audit_log`` section defines configuration parameters related to :ref:`audit logging <enterprise_audit_module>`.
+
+..  NOTE::
+
+    ``audit_log`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+
+* :ref:`audit_log.extract_key <configuration_reference_audit_extract_key>`
+* :ref:`audit_log.file <configuration_reference_audit_file>`
+* :ref:`audit_log.filter <configuration_reference_audit_filter>`
+* :ref:`audit_log.format <configuration_reference_audit_format>`
+* :ref:`audit_log.nonblock <configuration_reference_audit_nonblock>`
+* :ref:`audit_log.pipe <configuration_reference_audit_pipe>`
+* :ref:`audit_log.spaces <configuration_reference_audit_spaces>`
+* :ref:`audit_log.syslog_facility <configuration_reference_audit_syslog-facility>`
+* :ref:`audit_log.syslog_identity <configuration_reference_audit_syslog-identity>`
+* :ref:`audit_log.syslog_server <configuration_reference_audit_syslog-server>`
+* :ref:`audit_log.to <configuration_reference_audit_to>`
+
+..  _configuration_reference_audit_extract_key:
+
+..  confval:: audit_log.extract_key
+
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    If set to ``true``, the audit subsystem extracts and prints only the primary key instead of full
+    tuples in DML events (``space_insert``, ``space_replace``, ``space_delete``).
+    Otherwise, full tuples are logged.
+    The option may be useful in case tuples are big.
+
+    |
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_AUDIT_LOG_EXTRACT_KEY
+
+..  _configuration_reference_audit_file:
+
+..  confval:: audit_log.file
+
+    Specify a file for the audit log destination.
+    You can set the ``file`` type using the :ref:`audit_log.to <configuration_reference_audit_to>` option.
+    If you write logs to a file, Tarantool reopens the audit log at `SIGHUP <https://en.wikipedia.org/wiki/SIGHUP>`_.
+
+    |
+    | Type: string
+    | Default: 'var/log/{{ instance_name }}/audit.log'
+    | Environment variable: TT_AUDIT_LOG_FILE
+
+..  _configuration_reference_audit_filter:
+
+..  confval:: audit_log.filter
+
+    Enable logging for a specified subset of audit events.
+    This option accepts the following values:
+
+    *   Event names (for example, ``password_change``). For details, see :ref:`Audit log events <audit-log-events>`.
+    *   Event groups (for example, ``audit``).  For details, see :ref:`Event groups <audit-log-event-groups>`.
+
+    The option contains either one value from above or a combination of them.
+
+    To enable :ref:`user-defined audit log events <audit-log-custom>`, specify the ``custom`` value in this option.
+
+    |
+    | Type: array
+    | Possible values: 'all', 'audit', 'auth', 'priv', 'ddl', 'dml', 'data_operations', 'compatibility',
```

Comment:
Shouldn't we show that a possible value is, for example, `[audit, auth]`, not just `audit`? Now it looks like just one value can be set there.

One may also assume that the format is `audit,auth` -- a string of comma separated values.

Don't know, how to express it formally, but it seems, it should be clear from the reference documentation.

Reply:
There are examples for another options. I would add one here too.

Reply:
Added the example, updated the description a little.